sdk/core/core-paging/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.2.1 (Unreleased)
+
+### Other Changes
+
+- Updates package to work with the react native bundler. Browser APIs such as `URL` will still need to be pollyfilled for this package to run in react native. [PR #17783](https://github.com/Azure/azure-sdk-for-js/pull/17783)
+
```

Comment:
I believe this package does not reference `URL` or any browser APIs.

Reply:
You're right, I was thinking of `core-paging-rest`